packages/core/src/makeGetEndowmentsForConfig.js

```diff
@@ -14,8 +14,17 @@ function makeGetEndowmentsForConfig () {
     copyValueAtPath
   }
 
-  // for backwards compat only
-  function getEndowmentsForConfig (globalRef, config) {
+  /**
+   *
+   * @function getEndowmentsForConfig
+   * @param {object} sourceRef - Object from which to copy properties
+   * @param {object} targetRef - Object to which to copy properties
+   * @param {object} unwrapRef - For getters and setters, replaces targetRef as the 'this' value
+   * @param {object} config - LavaMoat package config
+   * @return {object} - The targetRef
+   *
+   */
+  function getEndowmentsForConfig (sourceRef, targetRef, unwrapRef, config) {
```

Comment:
rm `targetRef` from `getEndowmentsForConfig
` and just create a new obj for the `targetRef`
rm arg from everywhere `getEndowmentsForConfig` is called